packages/applet/package.json

```diff
@@ -13,15 +13,21 @@
   },
   "main": "./dist/index.cjs",
   "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
```

Comment:
I've removed it because:

![image](https://github.com/vuejs/devtools-next/assets/22515951/8c6c1eae-7aaf-43b2-8a16-bf2612e92ed6)